SwiftKit/src/main/java/org/swift/swiftkit/SwiftInstance.java

```diff
@@ -17,36 +17,87 @@
 import java.lang.foreign.GroupLayout;
 import java.lang.foreign.MemorySegment;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.Supplier;
 
-public interface SwiftInstance {
+public abstract class SwiftInstance {
```

Comment:
Now `SwiftInstance` is the base class for every imported Swift types. `SwiftValue` and `SwiftHeapObject` are interfaces mostly just for a marker at this point.

Reply:
Nice, thanks!